src/components/Article.jsx

```diff
@@ -1 +1,29 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, text, date }) => {
```

Comment:
So as I write above it's better to get the whole article here and destructure values inside the component
```suggestion
export const Article = ({ article }) => {
  const { title, text, date } = article;
```